packages/ethereum-storage/src/ethereum-storage.ts

```diff
@@ -56,6 +58,16 @@ export default class EthereumStorage implements StorageTypes.IStorage {
    */
   public maxIpfsReadRetry: number = getMaxIpfsReadRetry();
 
+  /**
+   * Timestamp of the dataId not mined on ethereum yet
+   */
+  private buffer: { [id: string]: number | undefined };
+
+  /**
+   * Url where can be reached the data hosted by this storage
+   */
+  private locationBufferUrl: string;
```

Comment:
can't find anywhere in the code where the location is actually used... don't get its purpose?